accounts/rpcserver.go

```diff
@@ -130,6 +130,28 @@ func (s *RPCServer) UpdateAccount(ctx context.Context,
 	return marshalAccount(account), nil
 }
 
+// CreditAccount increases the balance of an existing account in the account
+// database, by the given amount.
+func (s *RPCServer) CreditAccount(ctx context.Context,
+	req *litrpc.UpdateAccountBalanceRequest) (*litrpc.Account, error) {
+
+	log.Infof("[creditaccount] id=%s, label=%v, amount=%d",
+		req.Id, req.Label, req.Amount)
+
+	return nil, fmt.Errorf("not implemented")
+}
+
+// DebitAccount decreases the balance of an existing account in the account
+// database, by the given amount.
+func (s *RPCServer) DebitAccount(ctx context.Context,
+	req *litrpc.UpdateAccountBalanceRequest) (*litrpc.Account, error) {
+
+	log.Infof("[debitaccount] id=%s, label=%v, amount=%d",
+		req.Id, req.Label, req.Amount)
+
+	return nil, fmt.Errorf("not implemented")
+}
+
```

Comment:
this should not be needed in this commit. The RPC server embeds the UnimplementedAccountsServer

Reply:
Ah thanks! TIL